DSCResources/MSFT_xDatabase/MSFT_xDatabase.psm1

```diff
@@ -8,7 +8,7 @@ DacPacExtractionError=Extracting DacPac for Db failed, continuing with Dac Deplo
 '@
 }
 
-$SmoServerLocation = "${env:ProgramFiles(x86)}\Microsoft SQL Server\110\SDK\Assemblies\Microsoft.SqlServer.Smo.dll"
+Import-Module -DisableNameChecking $PSScriptRoot\..\xDatabase_Common
```

Comment:
This is because the various functions used have non-standard verbs. Ideally I'd change these but I think that's beyond the scope of this PR and is better done once they are consolidated into a single place. Is your suggestion though that it is better to do the import and have it notify us of the non-standard verbs?